src/components/ButtonMenu/buttonTrigger.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button';
+import RenderIf from '../RenderIf';
+
+export default function ButtonTrigger(props) {
+    const { label, icon, iconPosition, ...rest } = props;
+    const hasLeftIcon = iconPosition !== 'right';
+
+    const marginClass = hasLeftIcon ? 'rainbow-m-right_xx-small ' : 'rainbow-m-left_xx-small ';
+    const className =
+        icon && icon.props.className ? marginClass.concat(icon.props.className) : marginClass;
+    const iconElement = icon !== null ? React.cloneElement(icon, { className }) : null;
```

Comment:
This implies that the user should be responsible to pass an icon that have the className prop passed as className attr to its container. Since we should not depend on this let's add a wrapper element to the icon with the margin styles.
Also, note that we use styled-components for all our styles (https://styled-components.com/).